tests/multimodal/test_utils.py

```diff
@@ -92,18 +92,47 @@ def test_repeat_and_pad_placeholder_tokens(model):
     tokenizer = AutoTokenizer.from_pretrained(model)
 
     test_cases = [
-        ("<image>", 2, "<image><image>", [32000, 32000]),
-        ("<image><image>", 2, "<image><image><image>", [32000, 32000, 32000]),
+        ("<image>", 2, "<image><image>", [32000, 32000], [{
+            "offset": 0,
+            "length": 2
+        }]),
+        ("<image><image>", 2, "<image><image><image>", [32000, 32000,
+                                                        32000], [{
+                                                            "offset": 0,
+                                                            "length": 2
+                                                        }]),
         ("<image><image>", [3, 2], "<image><image><image><image><image>",
-         [32000, 32000, 32000, 32000, 32000]),
+         [32000, 32000, 32000, 32000, 32000], [{
+             "offset": 0,
+             "length": 3
+         }, {
+             "offset": 3,
+             "length": 2
+         }]),
         ("Image:<image>Image:<image>!", [3, 2],
          "Image:<image><image><image>Image:<image><image>!",
-         [9833, 28747, 32000, 32000, 32000, 9833, 28747, 32000, 32000, 918]),
-        ("<image>", [3, 2], "<image><image><image>", [32000, 32000, 32000]),
+         [9833, 28747, 32000, 32000, 32000, 9833, 28747, 32000, 32000, 918], [{
+             "offset":
+             2,
+             "length":
+             3
+         }, {
+             "offset":
+             7,
+             "length":
+             2
+         }]),
+        ("<image>", [3, 2], "<image><image><image>", [32000, 32000, 32000], [{
+            "offset":
+            0,
+            "length":
+            3
+        }]),
```

Comment:
This looks horrible, can we turn off the formatter here?

Reply:
TIL! Thank you for the suggestion.